src/main/java/hattrickdata/ExpandedCapacity.java

```diff
@@ -0,0 +1,21 @@
+package hattrickdata;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDateTime;
+
+@Getter
+@SuperBuilder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class ExpandedCapacity extends Capacity {
+
+    private LocalDateTime expansionDate;
```

Comment:
HODateTime

Reply:
Refactor: Review Finding 5: Use HODateTime instead of LocalDateTime